generator/src/main/java/com/scottlogic/deg/generator/restrictions/SetRestrictions.java

```diff
@@ -24,11 +22,21 @@
         return this.blacklist;
     }
 
+    private SetRestrictions(){
+        this.whitelist = null;
```

Comment:
Can you explain why we don't make an empty set here?

Reply:
In short, no - it would make for a nearer/safer implementation if we did.
It would take some reviewing of the generator code, which was outside of the scope of this PR however.

Reply:
OK cool thought so